File: pset1/mario/more/mario.c
# include <stdio.h>
# include <cs50.h>

int main(void)
{
    bool isValid = false;
    int x = 0;

    while (!isValid)
    {
        x = get_int("Height: ");
        if (x >= 0 & x < 24)
        {
            isValid = true;
        }
    }

    for (int i = 1; i <= x; i++)
    {
        for (int j = 1; j <= x - i; j++)
        {
            printf(" ");
        }

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }



}

File: pset6/mario/mario.py
from cs50 import get_int
x = get_int("Height: ")
while(x < 0 or x >= 24):
    x = get_int("Height: ")

for i in range(0, x):
    for j in range(0, x - i - 1):
        print(" ", end ="")
    for k in range(0, i + 1):
        print("#", end ="")
    print("  ", end ="")
    for l in range(0, i + 1):
        print("#", end ="")
    print("")